plugins/juju/README.md

```diff
@@ -0,0 +1,107 @@
+# juju plugin
+
+This plugin provides useful aliases and functions for [juju](https://juju.is/)
+(for TAB completion, refer to the 
+[official repo](https://github.com/juju/juju/blob/develop/etc/bash_completion.d/juju)).
+
+To use this plugin, add `juju` to the plugins array in your zshrc file.
+
+```zsh
+plugins=(... juju)
+```
+
+## Aliases
+Generally,
+- `!` means `--force --no-wait -y`
+- `ds` suffix means --destroy-storage
+
+### General
+| Alias | Command                                                | Description                                                               |
+|-------|--------------------------------------------------------|---------------------------------------------------------------------------|
+| jdl   | juju debug-log --ms                                    | Display log, with millisecond resolution                                  |
+| jdlr  | juju debug-log --ms --replay                           | Replay entire log                                                         |
+| jh    | juju help                                              | Show help on a command or other topic                                     |
+| jstj  | juju status --format=json                              | Show status in json format (more detailed)                                |
+| jst   | juju status --relations --storage --color              | Show status, including relations and storage, in color                    |
```

Comment:
Use `backticks` in the alias names and commands, and keep the table columns as fitted around the content as possible:
```suggestion
| `jdl`  | `juju debug-log --ms`                       | Display log, with millisecond resolution               |
| `jdlr` | `juju debug-log --ms --replay`              | Replay entire log                                      |
| `jh`   | `juju help`                                 | Show help on a command or other topic                  |
| `jstj` | `juju status --format=json`                 | Show status in json format (more detailed)             |
| `jst`  | `juju status --relations --storage --color` | Show status, including relations and storage, in color |
```